src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    <p className="Person__age"> {person.age ? `I am ${person.age}`:`` }</p>
```

Comment:
```suggestion
    <p className="Person__age">{person.age ? `I am ${person.age}`:`` }</p>
```

fix it everywhere
you don't need free space at the start of each row:
![image](https://user-images.githubusercontent.com/86923089/202892625-d6734e4d-0be6-48c7-92da-3b2ffad31b49.png)